doc/api/util.md

```diff
@@ -1024,6 +1024,227 @@ Otherwise, returns `false`.
 See [`assert.deepStrictEqual()`][] for more information about deep strict
 equality.
 
+## Class: `util.MIMEType`
+
+<!-- YAML
+added: REPLACEME
+-->
+
+> Stability: 1 - Experimental
+
+An implementation of [the MIMEType class](https://bmeck.github.io/node-proposal-mime-api/).
+
+In accordance with browser conventions, all properties of `MIMEType` objects
```

Comment:
Browsers have a deprecated global named `MimeType`: https://developer.mozilla.org/en-US/docs/Web/API/MimeType. I feel like we should at least mention that this isn’t that, that our version has a different API despite the very similar name (I assume?).

Reply:
IMHO that's not necessary, but if you make it a suggestion I can include it (or if you want to push a commit to that branch, that's also fine).